rules/autoconsent/postnl.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "postnl",
+    "vendorUrl": "https://postnl.nl",
+    "cosmetic": false,
+    "runContext": {
+        "urlPattern": "^https://([^.]+\\.)?postnl\\.nl/"
```

Comment:
Just for my own education, do we do this for site-specific rules where the selectors are pretty generic?

Reply:
yeah, exactly. When you're 100% sure this will only work on a specific site, there's no point checking the rule on other domains